dietpi/dietpi-drive_manager

```diff
@@ -490,6 +501,13 @@ aDRIVE_ISPARTITIONTABLE ${aDRIVE_ISPARTITIONTABLE[$i]}
 
 		rm $FP_TEMP_FSTAB
 
+		# Install needed file system write packages
+		if [[ $need_ntfs || $need_hfs ]]; then
+
+			G_AG_CHECK_INSTALL_PREREQ $need_ntfs $need_hfs
```

Comment:
Minor, these could probably be converted to 1 var?
```
pre_req_packages=''
pre_req_packages+='ntfs3g'
pre_req_packages+='htfs'
G_AG_CHECK_INSTALL_PREREQ $pre_req_packages
```

Reply:
@Fourdee 
Thought about the same first, but duplicates would be possible, as we loop though all drives. Would be needed to scan for already added strings and the overhead of this is larger than simply two variables, I though.

But I guess with some coffee, we could find an elegant way with one variable and no additional grep calls 😆.

Reply:
@MichaIng 

> But I guess with some coffee, we could find an elegant way with one variable and no additional grep calls 

lol 👍